### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'request'

### ISSUE THREAD

**Issue by jesusch** [barseghyanartur/django-mongoengine-filter#7]
django-filter has a different constructor for BaseFilterSet
https://github.com/carltongibson/django-filter/blob/master/django_filters/filterset.py#L190

I find it a bit odd as it's using '*' as parameter oO
My local solution for now was:

```
    def __init__(self, data=None, queryset=None, prefix=None, strict=None, *args, **kwargs):
        return super().__init__(data=None, queryset=None, prefix=None, strict=None)
```

**Comment by barseghyanartur** [comment]
For the reference, work on this is done in [sync-api](https://github.com/barseghyanartur/django-mongoengine-filter/tree/sync-api) branch.

Some refs (for clarification):

- https://github.com/carltongibson/django-filter/blob/efb9c7f63b69a1b0a09fb463309ba92522010c8e/django_filters/filterset.py
- https://github.com/carltongibson/django-filter/blob/master/django_filters/filterset.py

Any help on this is appreciated.

**Comment by oussjarrousse** [comment]
I believe this issue has been resolved in the 0.4.0 release in Dec 25, 2022. Current definition of BaseFilterSet.__init__() has the * separator, and is compatible with django-filter.

You will find the function definition here: https://github.com/barseghyanartur/django-mongoengine-filter/blob/master/django_mongoengine_filter/filterset.py#L189 
in the latest release

@barseghyanartur you can close this issue :-)

**Comment by oussjarrousse** [comment]
Very low hanging fruit!
